src/main/webapp/WEB-INF/jsp/preview.jsp

```diff
@@ -246,7 +246,7 @@
                     <!-- Email preview table -->
                     <table id="email-list-table" class="list-group" role="list">
                         <tr rsf:id="row:" class="email-row list-group-item" role="listitem">
-                           <td class="right-content-email-toggle">
+                           <td class="right-content-email-toggle" role="presentation">
```

Comment:
Fair enough, if it is a pain to migrate, keep the design as is.

> btw you can't zoom in email-list in google's Inbox native app, you can only zoom in the selected message

Is that on IOS? On Android I can zoom in and out on all views.